### PR TITLE
Feat/CC-883: Changes for the node v18/v20 upgrade

### PR DESCRIPTION
**Pull request by mwejuli-ch**
Upgrade changes for v18 and v20

### CODE REVIEW

File: test/middleware/session-middleware.test.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ describe("Session middleware", () => {
                 const sessionStore = Substitute.for<SessionStore>();
 
                 await SessionMiddleware(config, sessionStore, createSessionWhenNotFound)(request, Substitute.for<Response>(), nextFunction);
-
+                console.log(request);
```
Comment (hselvakumar) [inline test/middleware/session-middleware.test.ts:66]: do you still need this ? log the request ?

Comment (mwejuli-ch) [reply test/middleware/session-middleware.test.ts]: removed!

File: version [hunk 2]
```diff
@@ -1 +1 @@
-4.1
+5.0.0
```
Comment (hselvakumar) [inline version:1]: only 5.0 is needed here

Comment (mwejuli-ch) [reply version]: refactored!

**Review by hselvakumar — APPROVED** [review]

**Pull request merged**